pkg/reconciler/internal/diffutil/diff_util.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diffutil
```

Comment:
after checking with @joelanford following my thought about what needs to be checked here that it needs to be changed:
- Remove info logs that will gen duplicated entries
- Let's use this one for now just add diffutil and logs in the flags parses.